indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -164,9 +175,12 @@ class RHRegistrationsActionBase(RHManageRegFormBase):
 
     registration_query_options = ()
 
-    def _process_args(self):
+    @use_kwargs({
+        'registration_id': fields.List(fields.Integer(), missing=[]),
+    })
+    def _process_args(self, registration_id):
         RHManageRegFormBase._process_args(self)
-        ids = set(request.form.getlist('registration_id'))
+        ids = set(registration_id)
```

Comment:
I think we can safely skip this and just use it directly below. Duplicates in WHERE IN aren't a problem.